pyccel/errors/messages.py

```diff
@@ -8,6 +8,7 @@
 	that do not have any parameters.
 """
 
+ARRAYS_IN_MULTIPLE_RETURNS = "returning an array in multiple returns is not supported yet."
```

Comment:
This error is no longer needed

Reply:
Nor is this one:
https://github.com/pyccel/pyccel/blob/71ac69a8719fd3aa9b6f73241fd8a56703f297d4/pyccel/errors/messages.py#L34